src/main/java/com/jalasoft/webservice/model/ImageConvert.java

```diff
@@ -35,7 +35,8 @@ public Response convert(Criteria criteria) throws IOException {
 
         try {
 
```

Comment:
Normally there is not a space line after a statement.

Reply:
Solved